hm_1/tasks/task_04_02.py

```diff
@@ -0,0 +1,11 @@
+import urllib.request
+from collections import Counter
+
+
+def count_dots_on_i(url: str) -> int:
+    try:
+        with urllib.request.urlopen(url) as response:
+            html = str(response.read())
+        return Counter(html)["i"]
```

Comment:
Let's try to use a different approach.  What is the simplest algorithm to count `i`?